utils/rpms/daos.spec

```diff
@@ -58,6 +58,7 @@ Requires: protobuf-c
 Requires: spdk
 Requires: fio < 3.4
 Requires: openssl
+Requires: ndctl
```

Comment:
Is this needed for server, client or both?  If just server, it should move down between line 77 and 78.